File: design/circumference.py
#!/usr/bin/env python3
"""
Created by: Daniel Jeffrey
Created on: Feb 2025
This module calculates the circumfernce of a circle
    with a given diameter.
"""
import math


def main() -> None:
    x = float(input("What is the diameter of the circle in mm?"))
    "The main() function calculates the area of a 15 mm circle,"
    print("If a circle has a radius of ", x, " mm:")
    print(f"Circumference is {(math.pi * x):.5f} mm.")


if __name__ == "__main__":
    main()

File: design/circumference.c
// Copyright (c) 2025 D. Jeffrey All rights reserved
//
// Created by: D. Jeffrey
// Created on: Feb 2025
// This program calculates the circumference of a circle
//    with diameter

#include <stdio.h>
#include <math.h>

int main() {
    // this function calculates the area of a circle
    float diameter = printf("What is the diameter of the circle in mm?: \n");
    scanf("%f", &diameter);
    printf("If a circle has a radius of %.2f mm: \n", (diameter));
    printf("Circumference is %.5f mm.\n", (M_PI * diameter));
    return 0;
}
